Makefile

```diff
@@ -118,3 +118,9 @@ vendor-update:
 	glide update -v
 	# Vendors OpenShift and its dependencies
 	./scripts/vendor-openshift.sh
+
+# Test if the changed spec.go is valid and JSONSchema can be generated out of it
+.PHONY: test-jsonschema-generation
+test-jsonschema-generation:
+	sudo docker run -v `pwd`/pkg/spec/spec.go:/data/spec.go:ro,Z surajd/kedgeschema
```

Comment:
having anything "sudo" within Makefile is really bad...

Please remove sudo! If a user wishes to run test-jsonschema-generation they can use `make test-jsonschema-generation`

Reply:
resolved!